crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -735,7 +737,7 @@ where
         let slot_now = create_inherent_data_providers.slot();
 
         let checked_header = {
-            let pre_digest = find_pre_digest::<Block>(&block.header)?;
+            let pre_digest = find_pre_digest::<Block>(&block.header).map_err(subspace_err)?;
```

Comment:
Just a minor thing. Would it be better to make it a trait on `Result`? Because right now it looks more like mapping error, instead of just logging it. Something like this:
```rust
impl<T, E> LogResult for Result<T, E> {
    fn log_debug(&self, target: &'static str) {
        if let Err(e) = self {
            debug!(target: target, ?e, "Encountered error");
        }
    }
}
```

Reply:
I think we need to rename `subspace_err` in the future. It was inherited from BABE implementation, I think the idea there is to have all of the internal errors logged with consensus target, such that you can filter them all using target, even if they also result in other errors down the road. Not even sure we want to necessarily preserve it long-term, so I kept it for now, just refactored a bit.

Adding method to `Result` for this doesn't desirable to me.